### DISCUSSION TITLE
Publish java 8 compatible version of 8.2.RC2

### DISCUSSION THREAD

**Discussion by shipofphils** [vladimir-bukhtoyarov/bucket4j#345] [General]
Hello, I would like to use the Lettuce RedisClusterClient with bucket4j coming in 8.2, but currently stuck using Java 8.

Would you be able to publish the 8.2.RC2 version for java 8 to Maven Central?

Thank you.

**Comment by vladimir-bukhtoyarov** [comment]
Hello @shipofphils 

I will publish it at weekend.

**Comment by vladimir-bukhtoyarov** [comment]
Published.

**Reply by shipofphils** [reply]
Thank you @vladimir-bukhtoyarov !
